counsel.el

```diff
@@ -2796,7 +2805,8 @@ to custom."
       (while start-pos
         (let ((name (org-get-heading
                      (not counsel-org-goto-display-tags)
-                     (not counsel-org-goto-display-todo)))
+                     (not counsel-org-goto-display-todo)
+                     (not counsel-org-goto-display-priority)))
```

Comment:
`org-get-heading` only accepts two optional arguments in Org versions < 9.1.1, which was only released a couple of months ago.

Reply:
Would you opt for leaving it out rather, or do a version comparison?